.github/actions/build-manager-image/action.yaml

```diff
@@ -0,0 +1,60 @@
+name: 'Build Istio Manager Image'
+description: 'Builds Istio manager image'
+inputs:
+  operator-image-name:
+    description: 'Operator image name used for creation of Istio manager image'
+    required: true
+  platforms:
+    description: 'Platforms to build the image for (comma-separated), e.g. linux/amd64,linux/arm64,linux/arm/v7'
+    required: false
+    default: 'linux/amd64'
+  push-image:
+    description: 'Push the image to GCP'
+    required: false
+    default: 'false'
+  push-sa-key:
+    description: 'Service account key to push the image to GCP'
+    required: false
+    default: ''
+runs:
+  using: "composite"
+  steps:
+    - name: Set up QEMU
```

Comment:
We don't need it and I kept it, just if we want to use it in the future, since it only takes ~2 seconds. But I can remove it, if you think it is better to remove it. But then I'd also remove the platform arg and always build it for linux.